### PR TITLE
MLS - Changed the dashboard to match the design

### PR DESCRIPTION
**Pull request by llali**
Changed the dashboard to match the design. 
TODO:
- Not all the images and icons are ready so using duplicate, temporary images for now
- Need to setup "learn more" links to the actual page. right now nothing is available
- Couldn't set background color to what it was suggested in the design because the color looks back in dark theme. Have to figure out how to set different colors for dark/light themes

Screen Shot 2020-04-08 at 11 45 32 AM

### CODE REVIEW

File: extensions/machine-learning-services/package.json [hunk 1]
```diff
@@ -55,7 +55,11 @@
     "commands": [
       {
         "command": "mls.command.managePackages",
-        "title": "%mls.command.managePackages%"
+        "title": "%mls.command.managePackages%",
+        "icon": {
+          "light": "./images/installPackages.svg",
+          "dark": "./images/installPackages.svg"
```
Comment (udeeshagautam) [inline extensions/machine-learning-services/package.json:61]: dark light are same for now or now they always will be same?

File: extensions/machine-learning-services/src/common/constants.ts [hunk 2]
```diff
@@ -164,8 +167,15 @@ export const updateModelFailedError = localize('models.updateModelFailedError',
 export const importModelFailedError = localize('models.importModelFailedError', "Failed to register the model");
 export const loadModelParameterFailedError = localize('models.loadModelParameterFailedError', "Failed to load model parameters'");
 export const unsupportedModelParameterType = localize('models.unsupportedModelParameterType', "unsupported");
-
-
+export const dashboardTitle = localize('dashboardTitle', "SQL ML");
```
Comment (udeeshagautam) [inline extensions/machine-learning-services/src/common/constants.ts:170]: nit : SQL Machine Learning sounds clearer than just ML.. but if we have closed thats ok

File: extensions/machine-learning-services/src/controllers/mainController.ts [hunk 3]
```diff
@@ -110,13 +111,16 @@ export default class MainController implements vscode.Disposable {
 		let modelManagementController = new ModelManagementController(this._apiWrapper, this._rootPath,
 			azureModelsService, registeredModelService, predictService);
 
+		let dashboardWidget = new DashboardWidget(this._apiWrapper, this._rootPath);
+		dashboardWidget.register();
```
Comment (udeeshagautam) [inline extensions/machine-learning-services/src/controllers/mainController.ts:115]: this is using the new dashboard flow right? (looks so from the snapshot but just to confirm) did you have to do somehting extra there?

File: extensions/machine-learning-services/package.json (hunk 1 above)
Comment (llali) [reply extensions/machine-learning-services/package.json:61]: yes waiting for dark theme icons and colors

File: extensions/machine-learning-services/src/common/constants.ts (hunk 2 above)
Comment (llali) [reply extensions/machine-learning-services/src/common/constants.ts:170]: @MsSQLGirl is working on finalizing the name

File: extensions/machine-learning-services/src/controllers/mainController.ts (hunk 3 above)
Comment (llali) [reply extensions/machine-learning-services/src/controllers/mainController.ts:115]: no this registers the widget to show up in dashboard

**Pull request merged**